example_18-01.cpp

```diff
@@ -1,165 +1,171 @@
 // Example 18-1. Reading a chessboard’s width and height, reading and collecting
-// the
-// requested number of views, and calibrating the camera
-//
-// **GARY WILL MODIFY THIS TO ALSO READ FROM DISK**
-//
-#include <opencv2/opencv.hpp>
+// the requested number of views, and calibrating the camera
 #include <iostream>
+#include <opencv2/opencv.hpp>
+
+using std::vector;
+using std::cout;
+using std::cerr;
+using std::endl;
 
-using namespace std;
-
-void help(char **argv) {
-  cout << "\n\n"
-       << "Example 18-1:\nReading a chessboard’s width and height,\n"
-       << "              reading and collecting the requested number of views,\n" 
-       << "              and calibrating the camera\n\n" 
-       << "Call:\n" << argv[0] << " <board_width> <board_height> <number_of_boards> <ms_delay_framee_capture> <image_scaling_factor>\n\n"
-       << "Example:\n./example_18-01 9 6 15 500 0.5\n"
-	   << "\n -- use the checkerboard9x6.png provided\n\n"
-       << " * First it reads in checker boards and calibrates itself\n" 
-       << " * Then it saves and reloads the calibration matricies\n"
-       << " * Then it creates an undistortion map and finaly\n"
-       << " * It displays an undistorted image\n"
-       << endl;
+void help(char **argv) {  // todo rewrite this
+    cout << "\n\n"
+         << "Example 18-1:\nReading a chessboard’s width and height,\n"
+         << "              reading and collecting the requested number of views,\n" 
+         << "              and calibrating the camera\n\n" 
+         << "Call:\n" << argv[0] << " <board_width> <board_height> <number_of_boards> <if_video,_delay_between_framee_capture> <image_scaling_factor>\n\n"
+         << " * First it reads in checker boards and calibrates itself\n" 
```

Comment:
I'd rather leave in the explicit comment on how to run this:
"Example:\n./example_18-01 9 6 15 500 0.5\n"		 +         << " * Then it creates an undistortion map and finaly\n"
 -	   << "\n -- use the checkerboard9x6.png provided\n\n"

or 

"Example:\n" << argv[0] << " 9 6 15 500 0.5\n"		 +         << " * Then it creates an undistortion map and finaly\n"
 -	   << "\n -- use the checkerboard9x6.png provided\n\n"